Mathmagician.Tests/FibonacciTest.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class FibonacciTest
+    {
+        [TestMethod]
+        public void FibonacciIsAnObject()
+        {
+            Fibonacci fibonacci = new Fibonacci();
+            Assert.IsNotNull(fibonacci);
+
+        }
+        [TestMethod]
+        public void FibonacciStartsWithZero()
```

Comment:
We need to add a test to check whether the list count == numberToWrite (I think you're on that already).